browser/ui/views/infobars/brave_wayback_machine_infobar_contents_view.cc

```diff
@@ -45,7 +46,7 @@
 
 namespace {
 // IDs of the colors to use for infobar elements.
-constexpr int kInfoBarLabelBackgroundColor = ThemeProperties::COLOR_INFOBAR;
+constexpr int kInfoBarLabelBackgroundColor = kColorInfoBarBackground;
```

Comment:
As you change to color id, `GetColor()` for this should use color provider instead of theme provider.

Reply:
Thanks for noticing that. Fixed in da09360afcfa08d8d8304bf3bded0192c926be15.